src/transports/metaplaylist/pipelines.ts

```diff
@@ -156,7 +156,18 @@ export default function(options : ITransportOptions): ITransportPipelines {
       const url = response.url == null ? loaderURL :
                                          response.url;
       const { responseData } = response;
-      return handleParsedResult(parseMetaPlaylist(responseData, url));
+
+      const serverTimeOffset = options.serverSyncInfos ?
```

Comment:
Why is Smooth calculating the `serverTimeOffset` in the parser but MetaPlaylist in the parsers?